cudaaligner/samples/sample_cudaaligner.cpp

```diff
@@ -21,6 +21,7 @@
 #include <claraparabricks/genomeworks/utils/genomeutils.hpp>
 
 #include <cuda_runtime_api.h>
+#include <getopt.h>
```

Comment:
Why is this needed?

Reply:
Removed in 405ec7e4e0b090a3323da5970c038a219078c5bc. This was a compile error for me after a merge, but it looks like it was then included twice.